File: Server/EmployeeServer.php
<?php
function login($email, $password) {
  $conn=new mysqli("localhost", "root", "", "easyproject");
  $loginQuery="SELECT * FROM users WHERE email='" . $email . "' AND password='" . $password . "' AND isAdmin='0'";
  $result=mysqli_query($conn, $loginQuery);
  if (mysqli_num_rows($result) > 0) {
    $str="";
    foreach (mysqli_fetch_assoc($result) as $key => $value) {
      if ($key == "photo") {
        $path=new SplFileInfo($value);
        $str .= $key . ":" . $path->getRealPath() . ";";
      } else {
        $str .= $key . ":" . $value . ";";
      }
    }
    return $str;
  } else {
    return "Email or Password is incorrect";
  }
}

function updateInfo($id, $email, $password, $name, $address) {
  $conn=new mysqli("localhost", "root", "", "easyproject");
  $updateQuery="Select email from users where id='" . $id . "'";
  $result=mysqli_query($conn, $updateQuery);
  $row=mysqli_fetch_row($result)[0];
  
  if ($row == $email) { // if email is not changed
    $updateQuery="UPDATE users SET email='$email', password='$password', name='$name', address='$address', isAdmin='0' WHERE email='$email'";
    return (mysqli_query($conn, $updateQuery) ? "Updated Successfully" : "Updated Failed");
  } else { // if email is changed
    $emails=array();
    $getEmailsQuery="SELECT email FROM users";
    $result=mysqli_query($conn, $getEmailsQuery);
    while ($row1=mysqli_fetch_assoc($result)) {
      $emails[]=$row1['email'];
    }
    if (in_array($email, $emails)) { // if email is already exist
      return "Email is already taken!";
    } else { // if email is not exist
      $updateQuery="UPDATE users SET email='$email', password='$password', name='$name', address='$address', isAdmin='0' WHERE email='$row'";
      return (mysqli_query($conn, $updateQuery) ? "Updated Successfully" : "Updated Failed");
    }
  }
}

function updateImage($imagePath, $id, $photo) {
  $conn=new mysqli("localhost", "root", "", "easyproject");
  $updateQuery="Select photo from users where id='" . $id . "'";
  $result=mysqli_query($conn, $updateQuery);
  $row=mysqli_fetch_row($result)[0];
  unlink($row); // delete old photo
  file_put_contents($imagePath, base64_decode($photo)); // upload new photo
  $updateQuery="UPDATE users SET photo='$imagePath' WHERE id='$id'";
  return (mysqli_query($conn, $updateQuery) ? "Updated Successfully" : "Updated Failed");
}

switch ($_REQUEST['function']) {
  case "login":
    echo login($_REQUEST['email'], $_REQUEST['password']);
    break;
  case "updateInfo":
    echo updateInfo($_REQUEST['id'], $_REQUEST['email'], $_REQUEST['password'], $_REQUEST['name'], $_REQUEST['address']);
    break;
  case "updateImage":
    $imagePath='./Images/' . uniqid(rand(), true) . '.' . $_REQUEST['photoExtension'];
    echo updateImage($imagePath, $_REQUEST['id'], $_REQUEST['photo']);
    break;
  default:
    echo "Invalid Request";
}

File: Client/src/main/java/edu/najah/server/EmployeeServer.java
package edu.najah.server;

import java.io.FileOutputStream;
import java.nio.file.Path;
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.PreparedStatement;
import java.sql.ResultSet;
import java.util.Base64;
import java.util.UUID;

import static java.nio.file.Files.delete;

public class EmployeeServer {
  public static String login(String email, String password) {
    try {
      String result = "";
      Class.forName("com.mysql.cj.jdbc.Driver");
      Connection connection = DriverManager.getConnection("jdbc:mysql://localhost/easyproject", "root", "");
      PreparedStatement preparedStatement = connection.prepareStatement("SELECT * FROM users WHERE email='" + email + "' AND password='" + password + "' AND isAdmin='0'");
      ResultSet resultSet = preparedStatement.executeQuery();
      if (resultSet.next()) {
        result += "id:" + resultSet.getString("id") + ";";
        result += "email:" + resultSet.getString("email") + ";";
        result += "password:" + resultSet.getString("password") + ";";
        result += "name:" + resultSet.getString("name") + ";";
        result += "address:" + resultSet.getString("address") + ";";
        result += "photo:C:/xampp/htdocs/Server" + resultSet.getString("photo").substring(1) + ";";
        result += "isAdmin:" + resultSet.getString("isAdmin") + ";";
        return result;
      }
      return "Email or Password is incorrect";
    } catch (Exception e) {
      e.printStackTrace();
      return "Login Failed";
    }
  }
  
  public static String updateInfo(String id, String email, String password, String name, String address) {
    try {
      Class.forName("com.mysql.cj.jdbc.Driver");
      Connection connection = DriverManager.getConnection("jdbc:mysql://localhost/easyproject", "root", "");
      PreparedStatement preparedStatement = connection.prepareStatement("Select email from users where id='" + id + "'");
      ResultSet resultSet = preparedStatement.executeQuery();
  
      // check if email is not changed
      if (resultSet.next() && resultSet.getString("email").equals(email)) {
        preparedStatement = connection.prepareStatement("UPDATE users SET password='" + password + "', name='" + name + "', address='" + address + "' WHERE id='" + id + "'");
        return preparedStatement.executeUpdate() > 0 ? "Updated Successfully" : "Updated Failed";
      }
  
      // check if email is changed
      preparedStatement = connection.prepareStatement("Select email from users where email='" + email + "'");
      resultSet = preparedStatement.executeQuery();
      if (resultSet.next()) {
        return "Email is already taken!";
      }
      // if email is not exist
      preparedStatement = connection.prepareStatement("UPDATE users SET email='" + email + "', password='" + password + "', name='" + name + "', address='" + address + "' WHERE id='" + id + "'");
      return preparedStatement.executeUpdate() > 0 ? "Updated Successfully" : "Updated Failed";
    } catch (Exception e) {
      e.printStackTrace();
      return "Updated Failed";
    }
  }
  
  public static String updateImage(String photoExtension, String id, String photo) {
    Connection connection;
    PreparedStatement preparedStatement;
    ResultSet resultSet;
    try {
      String imagePath = "./Images/" + UUID.randomUUID() + "." + photoExtension;
      Class.forName("com.mysql.cj.jdbc.Driver");
      connection = DriverManager.getConnection("jdbc:mysql://localhost/easyproject", "root", "");
      preparedStatement = connection.prepareStatement("Select photo from users where id='" + id + "'");
      resultSet = preparedStatement.executeQuery();
      if (resultSet.next())
        delete(Path.of("C:/xampp/htdocs/Server" + resultSet.getString("photo").substring(1))); // unlink the old photo
      byte[] decodedImage = Base64.getDecoder().decode(photo);
      FileOutputStream fileOutputStream = new FileOutputStream("C:/xampp/htdocs/Server" + imagePath.substring(1));
      fileOutputStream.write(decodedImage);
      fileOutputStream.close();
      preparedStatement = connection.prepareStatement("UPDATE users SET photo='" + imagePath + "' WHERE id='" + id + "'");
      return preparedStatement.executeUpdate() > 0 ? "Updated Successfully" : "Updated Failed";
    } catch (Exception e) {
      e.printStackTrace();
      return "Login Failed";
    }
  }
}